Unable to pass the test for build under ArchLinux

```
./gradlew build && ./gradlew postbuild                          
:copy_plugin
:build_unityPackage
Packaging Complete!
:compile_IOSResolver
XBuild Engine Version 2.6.5.0
Mono, Version 2.6.5.0
Copyright (C) Marek Sieradzki 2005-2008, Novell 2008-2009.
/home/skad/Unity3D/unity-jar-resolver/source/JarResolver.sln:  warning : Don't know how to handle GlobalSection MonoDevelopProperties, Ignoring.

Build started 23/11/2017 14:31:14.
__________________________________________________
Project "/home/skad/Unity3D/unity-jar-resolver/source/JarResolver.sln" (IOSResolver target(s)):

	Target ValidateSolutionConfiguration:
		Building solution configuration "Debug|Any CPU".

	Target VersionHandler:
		Project "/home/skad/Unity3D/unity-jar-resolver/source/VersionHandler/VersionHandler.csproj" (default target(s)):
		
			Target _ValidateEssentialProperties:
		
			Target PrepareForBuild:
				Configuration: Debug Platform: AnyCPU
		
			Target SplitProjectReferencesByExistent:
		
			Target ResolveProjectReferences:
		
			Target ResolveAssemblyReferences:
		
			Target CopyFilesMarkedCopyLocal:
		
			Target AssignTargetPaths:
		
			Target SplitResourcesByCulture:
		
			Target CreateManifestResourceNames:
		
			Target GenerateResources:
		
			Target GenerateSatelliteAssemblies:
			Skipping target "GenerateSatelliteAssemblies" because its outputs are up-to-date.
		
			Target CoreCompile:
			Skipping target "CoreCompile" because its outputs are up-to-date.
		
			Target GetCopyToOutputDirectoryItems:
		
			Target DeployOutputFiles:
		
			Target _GetCompileOutputsForClean:
		
			Target _RecordCleanFile:
		
		Done building project "/home/skad/Unity3D/unity-jar-resolver/source/VersionHandler/VersionHandler.csproj".
.		
.
.
Done building project "/home/skad/Unity3D/unity-jar-resolver/source/JarResolver.sln".

Build succeeded.

Warnings:

/home/skad/Unity3D/unity-jar-resolver/source/JarResolver.sln:  warning : Don't know how to handle GlobalSection MonoDevelopProperties, Ignoring.

	 1 Warning(s)
	 0 Error(s)

Time Elapsed 00:00:00.2201320
:clean_pluginProjDir
:update_metadataForVersion UP-TO-DATE
:copy_pluginTemplate
:copy_manifestMetadata
:copy_IOSResolver
:copy_PlayServicesResolver
:copy_VersionHandler
:copy_VersionHandlerImpl
:inject_versionIntoMetaFiles
:generate_manifest
:test_packageManager
The assembly mscorlib.dll was not found or could not be loaded.
It should have been installed in the `/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/1234567890/lib/mono/1.0/mscorlib.dll' directory.
:test_packageManager FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test_packageManager'.
> Process 'command '/opt/Unity/Editor/Data/Mono/bin/nunit-console'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 11.132 secs
```

When working on #87 i was facing another bug
I have mono / nunit-console / unity installed.

--- a/build.gradle
+++ b/build.gradle
@@ -116,12 +116,9 @@ project.ext {
     if (os_windows) {
         nunit_console_exe = findUnityPath("NUNIT_CONSOLE_EXE", false,
             fileTree(dir: unity_root).matching { include '**/Mono/bin/nunit-console2.bat'})
-    } else if (os_osx) {
+    } else if (os_osx || os_linux) {
         nunit_console_exe = findUnityPath("NUNIT_CONSOLE_EXE", false,
             fileTree(dir: unity_root).matching { include '**/nunit-console2'})
-    } else if (os_linux) {
-        nunit_console_exe = findUnityPath("NUNIT_CONSOLE_EXE", false,
-            fileTree(dir: unity_root).matching { include '**/nunit-console'})
     }
     if (nunit_console_exe == null || !nunit_console_exe.exists()) {
         logger.warn("nunit_console command not found, compilation may fail.")

